website/en/docs/types/arrays.md

```diff
@@ -116,3 +116,27 @@ if (value !== undefined) {
 
 As Flow is made to be smarter it may be possible in the future to fix this
 problem, but for now you should be aware of it.
+
+## `$ReadOnlyArray<T>` <a class="toc" id="toc-readonlyarray" href="#toc-readonlyarray"></a>
+
+Similar to `$ReadOnly<T>`, it is the supertype of all arrays and all tuples and
```

Comment:
Make `$ReadOnly<T>` a link to https://flow.org/en/docs/types/utilities/#toc-readonly.